components/ai_chat/resources/page/components/filter_menu/filter_menu.tsx

```diff
@@ -52,6 +54,7 @@ export function MatchedText(props: { text: string; match?: Match }) {
 }
 
 export default function FilterMenu<T>(props: Props<T>) {
+  const domToItem = React.useRef(new Map<HTMLElement, T>())
```

Comment:
This function could do with a comment especially explaining what role `T` plays, that might help explain what `domToItem` is storing - otherwise that needs a comment because I'm not sure the variable name is self explanatory.

Reply:
I've simplified this a bit - I'll add the comment as a separate PR